src/config.json

```diff
@@ -961,6 +961,171 @@
     "ipfs.io"
   ],
   "blacklist": [
+    "blur-assets.com",
+    "pudgypengiuns.xyz",
+    "aptos-drop.org",
+    "digiidaigaku.com",
+    "sui-foundation.com",
+    "blurcoin.net",
+    "bluronline.xyz",
+    "blur-airdrop.vip",
+    "ea-blur.site",
+    "blurlo.xyz",
+    "blur-prize.com",
+    "blurmarket.net",
+    "blurx.cfd",
+    "sui-coin.com",
+    "blurxdrop.space",
+    "intotheblur.xyz",
+    "blur-yacht.com",
+    "blur-community.com",
+    "blur-season.com",
+    "tokens-blur.com",
+    "blur-overwiev.net",
+    "martianwallet.su",
+    "blur-community.org",
+    "blure-official.net",
+    "blurs.site",
+    "blurnft.club",
+    "wwwaptoslabs.com",
+    "dao-aptoslabs.com",
+    "sui-drop.org",
+    "blurairdrop.su",
+    "blure.pro",
+    "opersea-io.info",
+    "es-blur.site",
+    "blur-apecoin.com",
+    "claim-web3.xyz",
+    "bluri.org",
+    "opengiweaway.com",
+    "aptosmadeeasy.com",
+    "ethdenver-festival.com",
+    "coinblur.com",
+    "operseas.org",
+    "looksrare-token.org",
+    "aptos-network.org",
+    "ethdenver-festival.org",
+    "opense.su",
+    "eth-denver-event.com",
+    "eth-denver-event.org",
+    "eihdenver.com",
+    "digikagaku.net",
+    "pudgy-penguins-discover.com",
+    "pudgy-penguins-discover.org",
+    "reglster-pudgypenguin.com",
+    "reglster-doodles2.app",
+    "blur-portfolio.org",
+    "map-doodles.app",
+    "betas-doodles.app",
+    "blur-tokens.xyz",
+    "pudgypenguins-chain.com",
+    "aptos-ai.com",
+    "blurairdrop.org",
+    "loouksrare.com",
+    "suinetwork.site",
+    "box-doodles.app",
+    "nft-blur.xyz",
+    "blure.site",
+    "nftblur.pro",
+    "blur.gifts",
+    "zapper-app.com",
+    "project-blur.com",
+    "blurapp.net",
+    "blur-jo.com",
+    "transactions.openseail.pw",
+    "digidaingaku.com",
+    "registers-blur.com",
+    "blur-bounty.xyz",
+    "blur-air.org",
+    "blurclaims.xyz",
+    "tokens-blur.org",
+    "nft-blur.org",
+    "blur-air.pro",
+    "blurio-airdrop.xyz",
+    "magicedem.app",
+    "pudgypenguins-chain.org",
+    "aptosretrodrop.site",
+    "spaceportdidaigaku.pro",
+    "zksyncs.app",
```

Comment:
confused why these are showing up in the diff too, if they are already part of main